app/src/main/java/ru/prometeydev/movie/ui/movieslist/MoviesListFragment.kt

```diff
@@ -51,8 +82,10 @@ class MoviesListFragment : BaseFragment() {
 
     @Suppress("UNCHECKED_CAST")
     override fun <T> bindViews(data: T) {
```

Comment:
Why Generics here?

Reply:
Because in MovieDetailsFragment in bindView we pass the MovieDetails type, and in MovieListFragment we pass PagingData <Movie>.